Tools/autotest/pysim/util.py

```diff
@@ -309,7 +325,9 @@ def start_SITL(binary,
         f.write("settings set target.process.stop-on-exec false\n")
         f.write("process launch\n")
         f.close()
-        if os.environ.get('DISPLAY'):
+        if sys.platform == "darwin" and os.getenv('DISPLAY'):
+            cmd.extend(['lldb', '-s', '/tmp/x.lldb', '--'])
```

Comment:
Why aren't we starting this in a new graphical terminal?

Reply:
we do that down below https://github.com/ArduPilot/ardupilot/pull/14363/files/af2fbe4047435c3bdb3bcc987fd3ce92cb18b865#diff-d2de282cca7753bde1b8719902d06dcbR355